tests/FINDOLOGIC/Export/Tests/CSVSerializationTest.php

```diff
@@ -102,6 +102,19 @@ public function testMinimalItemIsExported()
         $this->assertInternalType('string', $export);
     }
 
+    public function testHeadingIsOnlyWrittenForFirstPage()
+    {
+        $item = $this->getMinimalItem();
+        $export = $this->exporter->serializeItems([$item], 0, 1, 2);
+
+        $this->assertContains(CSVExporter::HEADING, $export);
+
+        $item = $this->getMinimalItem();
```

Comment:
This line is obsolete as we already have the minimal item on line 107

Reply:
This is intentional, because we wouldn't export the same item in two pages. Technically, it probably doesn't matter, but it's more realistic like this.